code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -99,8 +99,22 @@
 	name = "Santa's hat"
 	desc = "Ho ho ho. Merrry X-mas!"
 	icon_state = "santahat"
+	species_fit = list("Grey, Drask")
+	sprite_sheets = list(
+		"Grey" = 'icons/mob/species/Grey/head.dmi',
+		"Drask" = 'icons/mob/species/Drask/helmet.dmi'
+		)
 	flags = BLOCKHAIR | STOPSPRESSUREDMAGE
 	flags_cover = HEADCOVERSEYES
+/obj/item/clothing/head/helmet/space/santahat/attack_self(mob/user as mob)
+	if(src.icon_state == "santahat")

```

Comment:
`src `is implied, you don't need to put it infront - you can just do `if(icon_state == "santahat")` and the same for all the `srcs `below